### ISSUE TITLE
Error removing product

### ISSUE THREAD

**Issue by patrick-bigbridge** [clerkio/clerk-magento2#4]
While removing a product in the backend, I encounter this issue:

Exception #0 (Exception): Warning: join(): Invalid arguments passed in .../public_html/vendor/clerk/magento2/Model/Api.php on line 67

In Model/Api.php the function removeProduct() is written as if it expects an array, while it is only called once in the extension, with a scalar product id. Please change

```
    /**
     * Remove product
     *
     * @param $productIds
     */
    public function removeProduct($productIds)
    {
        $params = [
            'key'          => $this->_scopeConfig->getValue(Config::XML_PATH_PUBLIC_KEY),
            'private_key'  => $this->_scopeConfig->getValue(Config::XML_PATH_PRIVATE_KEY),
            'products'     => join(',', $productIds),
        ];

        $this->get('remove', $params);
    }
```

to

```
    /**
     * Remove product
     *
     * @param int $productId
     */
    public function removeProduct($productId)
    {
        $params = [
            'key'          => $this->_scopeConfig->getValue(Config::XML_PATH_PUBLIC_KEY),
            'private_key'  => $this->_scopeConfig->getValue(Config::XML_PATH_PRIVATE_KEY),
            'products'     => $productId,
        ];

        $this->get('remove', $params);
    }
```

**Comment by RBech** [comment]
@patrick-bigbridge  This is fixed in the next release too, see #2

**Comment by patrick-bigbridge** [comment]
Thanks, but please consider applying my change as well, and changing the name of the argument to $productId.

**Comment by RBech** [comment]
Sure, can do for clarity ;)

**Comment by RBech** [comment]
Fixed in 2.0.0
